lib/docker-utils.ts

```diff
@@ -0,0 +1,472 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import Docker = require('docker-toolbelt');
+import * as Promise from 'bluebird';
+import * as JSONStream from 'jsonstream';
```

Comment:
This appears to be causing issues with pkg (standalone builds), using this casing fixes:

`import * as JSONStream from 'JSONStream';`

Reply:
@srlowe Any idea how to get Typescript to find the types correctly when using the mixed casing? On my machine it's not able to find the types...

Reply:
@LucianBuzzo I'm not sure, but will have a look. I had only tested this by changing the js in the node_modules folder in the balena-cli PR that was failing.

Reply:
@srlowe What is the test for standalone builds actually checking when it fails due to the upper-casing here?

Reply:
I did a re-export of the module with the correct casing and it seems to make the compiler happy